alvr/server/cpp/alvr_server/HMD.h

```diff
@@ -54,10 +58,12 @@ class Hmd : public TrackedDevice, public vr::ITrackedDeviceServerDriver, vr::IVR
     virtual void
     GetProjectionRaw(vr::EVREye eEye, float *pfLeft, float *pfRight, float *pfTop, float *pfBottom);
     virtual vr::DistortionCoordinates_t ComputeDistortion(vr::EVREye eEye, float fU, float fV);
+    // TODO: Implement?
+    virtual bool ComputeInverseDistortion(vr::HmdVector2_t *pResult, vr::EVREye eEye, uint32_t unChannel, float fU, float fV) {}
 
     vr::VRInputComponentHandle_t m_proximity;
 
-    std::shared_ptr<CEncoder> m_encoder;
+    // std::shared_ptr<CEncoder> m_encoder;
```

Comment:
I need to check to see if windows code will be happy with this

Reply:
It most definitely isn't, but I also definitely haven't found a good way to do this routing stuff in HMD in general

Reply:
Honestly the windows code might need some unfuckery too